integration-cli/docker_cli_version_test.go

```diff
@@ -44,9 +43,7 @@ func testVersionPlatform(c *check.C, platform string) {
 	expected := "OS/Arch:      " + platform
 
 	split := strings.Split(out, "\n")
-	if len(split) < 14 { // To avoid invalid indexing in loop below
-		c.Errorf("got %d lines from version", len(split))
-	}
+	c.Assert(len(split) < 14, checker.Equals, false, check.Commentf("got %d lines from version", len(split)))
```

Comment:
Please adjust the logic to assert `true` rather than `false`
Might be awesome to have a `GreaterThan` `LessThan` checker.